python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -61,18 +65,24 @@ def is_deepspeed_available():
     from auto_gptq.utils.peft_utils import QuantLinearCuda, QuantLinearCudaOld
 
 
+if is_auto_awq_available():
+    from bigdl.llm.transformers.awq.linear import WQLinear_GEMM
+
+
 def is_linear_module(module):
 
     in_features = None
     out_features = None
     mp_group = None
 
+    is_awq = is_auto_awq_available() and isinstance(module, WQLinear_GEMM)
+
     if is_auto_gptq_available() and isinstance(module, QuantLinearCudaOld):
         in_features = module.infeatures
         out_features = module.outfeatures
         mp_group = None
         result = True
-    elif isinstance(module, nn.Linear):
+    elif isinstance(module, nn.Linear) or is_awq:
         in_features = module.in_features
         out_features = module.out_features
         mp_group = None
```

Comment:
Combine above into below?
```python
if isinstance(module, nn.Linear) or is_gptq or is_awq
```

Reply:
The names of in/out features in the gptq QuantLinearCudaOld are different from linear and awq, so we cannot combine them here.